js/models/conversations.js

```diff
@@ -273,7 +273,8 @@
       const recipientId = this.isPrivate() ? this.id : null;
       const groupNumbers = this.getRecipients();
 
-      const sendOptions = this.getSendOptions();
+      let sendOptions = this.getSendOptions();
```

Comment:
javascript... 🤦‍♂ 
You can use `const` even if you mutate the object attributes, as long as you don't overwrite the object itself.

Reply:
updated